src/main/java/org/broadinstitute/hellbender/tools/spark/utils/LargeLongBloomFilter.java

```diff
@@ -0,0 +1,191 @@
+package org.broadinstitute.hellbender.tools.spark.utils;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.spark.sv.SVUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * Set of long's that is too large to fit in a single LongBloomFilter. Works by binning each entry into a
+ * corresponding LongBloomFilter partition using the entry hash value. Partitioning is handled internally given
+ * a maximum partition size, number of elements to be added, and false positive probability.
+ */
+@DefaultSerializer(LargeLongBloomFilter.Serializer.class)
+public class LargeLongBloomFilter implements QueryableLongSet {
+
+    private final transient Logger logger = LogManager.getLogger(this.getClass());
+    private List<LongBloomFilter> filters;
```

Comment:
I don't see any reason that this couldn't be an array rather than a list.  Size is fixed for all time.